includes/admin/tools/class-settings-import.php

```diff
@@ -0,0 +1,681 @@
+                   value="<?php echo Give_Settings_Import::get_step(); ?>"/>
+			<?php
+		}
+
+		/**
+		 * Print the HTML for importer.
+		 *
+		 * @since 1.2
+		 */
+		static function html() {
+			$step = Give_Settings_Import::get_step();
+			?>
+            <section>
+                <table class="widefat export-options-table give-table <?php echo 'step-' . $step; ?>"
+                       id="<?php echo 'step-' . $step; ?>">
+                    <tbody>
+					<?php
+					if ( 1 === $step ) {
+						// Get the html of CSV file upload.
+						Give_Settings_Import::render_media_csv();
+					} elseif ( 2 === $step ) {
+						Give_Settings_Import::render_dropdown();
+					} elseif ( 3 === $step ) {
+						// Drop down for importer files.
+						Give_Settings_Import::start_import();
+					} elseif ( 4 === $step ) {
+						// Successful or fail message.
+						Give_Settings_Import::import_success();
+					}
+
+					if ( self::check_for_dropdown_or_import() == false ) {
+						$step = Give_Settings_Import::get_step();
+						?>
+                        <tr valign="top">
+                            <th></th>
+                            <th>
+                                <input type="submit"
+                                       class="button button-primary button-large button-secondary <?php echo 'step-' . $step; ?>"
+                                       id="recount-stats-submit"
+                                       value="<?php esc_attr_e( 'Submit', 'give' ); ?>"/>
+                            </th>
+                        </tr>
+						<?php
+					}
+					?>
+                    </tbody>
+                </table>
+            </section>
+			<?php
+		}
+
+		static function import_success() {
+			$report  = give_import_donation_report();
+			$total   = (int) $_GET['total'];
+			$total   = $total - 1;
+			$success = (bool) $_GET['success'];
+			?>
+            <tr valign="top" class="give-import-dropdown">
+                <th colspan="2">
+                    <h2>
+						<?php
+						if ( $success ) {
+							echo sprintf( __( 'Import complete! %s donations imported', 'give' ), '<strong>' . $total . '</strong>' );
+						} else {
+							echo sprintf( __( 'Failed to import %s donations', 'give' ), '<strong>' . $total . '</strong>' );
+						}
+						?>
+                    </h2>
+
+					<?php
+					$text      = __( 'Import Donation', 'give' );
+					$query_arg = array(
+						'post_type' => 'give_forms',
+						'page'      => 'give-tools',
+						'tab'       => 'import',
+					);
+					if ( $success ) {
+						$query_arg = array(
+							'post_type' => 'give_forms',
+							'page'      => 'give-payment-history',
+						);
+						$text      = __( 'View Donations', 'give' );
+					}
+					?>
+					<?php
+					if ( ! empty( $report['duplicate_donor'] ) ) {
+						?>
+                        <p>
+							<?php echo esc_html( wp_sprintf( '%s duplicate donors detected', $report['duplicate_donor'] ), 'give' ); ?>
```

Comment:
@raftaar1191 Please check this implementation. Fix them all.
<img width="946" alt="screen shot 2017-08-29 at 2 28 56 pm" src="https://user-images.githubusercontent.com/1784821/29813066-6b0e49d8-8cc6-11e7-944f-f134155c5bd4.png">

Reply:
Done